easy-ui-react/src/ModeSwitcher/ModeSwitcher.module.scss

```diff
@@ -0,0 +1,29 @@
+@use "../styles/common" as *;
+@use "../styles/unstyled";
+
+.trigger {
+  @include unstyled.button;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  gap: design-token("space.1");
+  padding: design-token("space.2");
+  border: design-token("shape.border_width.1") solid
+    theme-token("color.neutral.300");
+  border-radius: design-token("shape.border_radius.md");
+  cursor: pointer;
+  width: 100%;
+  max-width: 160px;
+}
+
+.triggerPopoverOpen {
+  border-color: theme-token("color.neutral.800");
+}
+
+.popover {
+  border: design-token("shape.border_width.1") solid
+    theme-token("color.neutral.300");
+  border-radius: design-token("shape.border_radius.md");
+  background-color: white;
+  padding: design-token("space.2");
+}
```

Comment:
there are some open design questions i have regarding the yellow color thats given in figma and what we have available in easy-ui and also the spacing within the popover / radio items